### PR TITLE
Prepare next release 2.4

### LINKED ISSUE #339: Release a new version on PyPi
I've already opened the PR #338 to update the changelog before the release of a new version of the package on pypi.

### PR DESCRIPTION
**Pull request by pauloxnet**
Fix #339

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -6,9 +6,12 @@ Changelog
 unreleased
 ^^^^^^^^^^
 
+v2.4 (2022-08-24)
+^^^^^^^^^^^^^^^^^
+
 - Add compatibility with Django 4.1
-- Drop compatibility for Django 2.2, 3.1
-- Drop compatibility for Python 3.6
+- **BACKWARD INCOMPATIBLE** Drop compatibility for Django 2.2, 3.1
```
Comment (michael-k) [inline docs/changes.rst:13]:
```suggestion
- **BACKWARD INCOMPATIBLE** Drop compatibility for Django < 3.2
```

**Review by michael-k — APPROVED** [review]
LGTM

**Review by blueyed — APPROVED** [review]
LGTM also, but the changelog suggestion makes sense.

From what I can tell it only updates the meta information (https://github.com/jazzband/django-configurations/pull/337).

**Comment by pauloxnet** [comment]
Thanks for the suggestions, I've updated this PR

**Review by jezdez — APPROVED** [review]

**Comment by pauloxnet** [comment]
@jezdez @blueyed @michael-k 
Given your approvals I merged this PR.
How will the new version now be released?

**Comment by michael-k** [comment]
There's a [release workflow](https://github.com/jazzband/django-configurations/blob/master/.github/workflows/release.yml).  Pushing the appropriate tag should be sufficient.

**Comment by pauloxnet** [comment]
> There's a [release workflow](https://github.com/jazzband/django-configurations/blob/master/.github/workflows/release.yml). Pushing the appropriate tag should be sufficient.

@michael-k I created the new tag
https://github.com/jazzband/django-configurations/releases/tag/2.4

**Comment by jezdez** [comment]
> According to https://jazzband.co/projects/django-configurations @jezdez and @blueyed are the leads on this project and are responsible for [uploading releases to PyPI](https://jazzband.co/about/releases#security).

Done!

**Pull request merged**